tutorials/cassandraOutput/src/test/resources/properties-CassandraOutputTestApp.xml

```diff
@@ -0,0 +1,34 @@
+<configuration>
```

Comment:
Do we need this conf file at both places? It's also there in  tutorials/cassandraOutput/src/site/conf/properties-CassandraOutputTestApp.xml